src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -844,16 +843,6 @@ function ComposerWithSuggestions(
                 resetKeyboardInput={resetKeyboardInput}
             />
 
-            {isValidReportIDFromPath(reportID) && (
-                <SilentCommentUpdater
```

Comment:
SilentCommentUpdater was introduced in order to avoid subscribing ComposerWithSuggestions to the draft state in onyx. Not only is this (imho) an anti-pattern, but ComposerWithSuggestions was re-subscribed to draft state anyway a long time ago, so we simply don't need this anymore. Debouncing the draft updates (which we were already doing) provides the performance that this was originally designed to do, but in a more idiomatic and straightforward way